pkg/sidecar/config/config.go

```diff
@@ -296,6 +302,21 @@ func (s *ScyllaConfig) validateCpuSet(ctx context.Context, cpusAllowed string, s
 	return nil
 }
 
+func setupIOTuneCache() error {
+	if _, err := os.Stat(scyllaIOPropertiesPath); err != nil {
+		if !errors.Is(err, os.ErrNotExist) {
+			return err
+		}
+
+		cachePath := path.Join(naming.DataDir, naming.ScyllaIOPropertiesName)
+		if err := os.Symlink(cachePath, scyllaIOPropertiesPath); err != nil {
+			return fmt.Errorf("can't create symlink from %q to %q: %w", scyllaIOPropertiesPath, cachePath, err)
+		}
+	}
```

Comment:
in the else branch we should log that the file already exists at V2

Reply:
added